src/test/java/gov/healthit/chpl/aqa/pageObjects/BasePage.java

```diff
@@ -59,4 +61,14 @@ public static WebElement headerNav(final WebDriver driver) {
     public static WebElement pageTitle(final WebDriver driver, final String pageTitle) {
         return driver.findElement(By.xpath("//h1[contains(text(),'" + pageTitle + "')]"));
     }
+
+    public static List<WebElement> getRowElements(final WebElement table) {
+        List<WebElement> rows = table.findElements(By.tagName("tr"));
+        return rows;
+    }
+
+    public static List<WebElement> getColElements(final WebElement table) {
```

Comment:
This method is called with `List<WebElement> colElts = ScheduledJobsPage.getColElements(col);` but the variable in the parameter is `table`

I don't think "col" is correct, but that's for a different comment. Here the problem is the "table"

Reply:
I'm not sure whether the changes that I pushed now looks good. If not, can you please suggest the names?